### ISSUE TITLE
incorrect simplification of abs((a + b*%i)^2) with a, b real

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#1287]
Imported from SourceForge on 2024-07-04 08:49:11
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2015-02-05 03:24:14
Original: https://sourceforge.net/p/maxima/bugs/2901

---

As discussed on mailing list 2015-02-04:

~~~~
(%i1) declare ([a, b], real);
(%o1)                                done
(%i2) abs((a + b*%i)^2);
                                            2
(%o2)                             (%i b + a)
~~~~

Note also:

~~~~
(%i7) abs(a + b*%i);
(%o7)                            abs(%i b + a)
(%i8) abs(a + b*%i)^2;
                                            2
(%o8)                             (%i b + a)
~~~~

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-04 08:49:13
Created by **[aleksasd](https://sourceforge.net/u/aleksasd/)** on 2015-02-09 15:05:37
Original: https://sourceforge.net/p/maxima/bugs/2901/#edff

---

use cabs

(%i1) cabs((a + b*%i)^2);
(%o1) b^2+a^2

(%i2) cabs(a + b*%i);
(%o2) sqrt(b^2+a^2)

We need to correct help for abs:
"The abs function represents the mathematical absolute value function and works for both numerical and symbolic values. If the argument, z, is a real or complex number, abs returns the absolute value of z."

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-04 08:49:17
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2017-08-20 05:07:19
Original: https://sourceforge.net/p/maxima/bugs/2901/#839d

---

- **status**: open --> closed

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-04 08:49:20
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2017-08-20 05:07:19
Original: https://sourceforge.net/p/maxima/bugs/2901/#cbce

---

Fixed by commit 03d2af28. Closing this report.
